src/sentry/models/event.py

```diff
@@ -613,13 +606,6 @@ class Event(EventCommon, Model):
     platform = models.CharField(max_length=64, null=True)
     datetime = models.DateTimeField(default=timezone.now, db_index=True)
     time_spent = BoundedIntegerField(null=True)
-    data = NodeField(
```

Comment:
We are not going to delete nodes when we delete an event with this PR, right ?
This was happening here:
https://github.com/getsentry/sentry/blob/master/src/sentry/db/models/fields/node.py#L179-L184

If I did not get this wrong, we need to find an intermediate solution before the synchronized consumer solution is ready.

Reply:
Good point. I think the easiest solution is to update deletions to use generate_node_id method instead of this field first. I'm worried that changing the event model to keep this as well as the new data field we are adding here might affect plugins and other things downstream that would get an event object with a slightly different shape than before.